openlibrary/i18n/cs/messages.po

```diff
@@ -14,7 +14,7 @@ msgstr ""
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: cs <LL@li.org>\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=utf-8\n"
+"content-type: text/plain; charset=utf-8\n"
```

Comment:
I don't think that `Content-Type` should be changed in any of the `.po` files.  This header entry is used by `gettext`, and I can't find any guidance regarding case sensitivity of header entries.  Better safe than sorry.